openstack/sharedfilesystems/apiversions/errors.go

```diff
@@ -0,0 +1,23 @@
+package apiversions
+
+import (
+	"fmt"
+)
+
+// ErrVersionNotFound is the error when the requested API version
+// could not be found.
+type ErrVersionNotFound struct{}
+
+func (e ErrVersionNotFound) Error() string {
```

Comment:
There is a kind of [framework](https://github.com/gophercloud/gophercloud/blob/master/errors.go) for errors in the gophercloud library.
This framework is used for defining errors, e.g. in https://github.com/gophercloud/gophercloud/blob/master/openstack/errors.go

However, [the framework](https://github.com/gophercloud/gophercloud/blob/master/errors.go) is not used here so it feels inconsistent.